src/test/java/net/snowflake/client/jdbc/ResultSetIT.java

```diff
@@ -1014,4 +1014,16 @@ public void testNextNegative() throws SQLException {
       rs.close();
     }
   }
+
+  @Test
+  public void testGeographyColumnType() throws Exception {
+    try (Connection conn = init()) {
+      conn.createStatement()
```

Comment:
let's add statement to try

Reply:
Removed the test as it was already covered by another one.